setup.py

```diff
@@ -8,17 +9,28 @@
 
 PACKAGE_NAME = "pylero"
 CLI_NAME = "pylero-cmd"
+RELEASE_FILE = "/etc/redhat-release"
+CHECK_VERSION_REGEX = re.compile(r'Fedora release (\d{1,2}).*')
 
 # change the data dir to be the etc dir under the package dir
 for scheme in list(INSTALL_SCHEMES.values()):
     scheme['data'] = '%s/%s/etc' % (scheme['purelib'], PACKAGE_NAME)
 
 install_requires_ = [
-    'suds-py3',
+    'suds-community',
     'click',
-    'pre-commit',
 ]
 
+# Update install_requires_ for feodra 36 and greater
+if os.path.exists(RELEASE_FILE):
+    with open(RELEASE_FILE) as version_file:
+        version_file_content = version_file.read()
+        if ("Fedora" in version_file_content) and (
+                int(CHECK_VERSION_REGEX.match(
+                    version_file_content).groups()[0]) > 35):
+            install_requires_ = ['suds', 'click']
```

Comment:
- nit, this actually creates a new obj 😅
- can we have a bool and append appropriate `suds` pkg after this block?

Reply:
ok